web/client/components/widgets/widget/ChartWidget.jsx

```diff
@@ -22,7 +22,7 @@ const renderHeaderLeftTopItem = ({ showTable, toggleTableView = () => {}} = {})
 };
 
 
-module.exports = ({
+export default ({
```

Comment:
i would avoid in general this, better to export a named function

Reply:
I used const but I think is the same